### PR TITLE
Unpickling problem test

### PR DESCRIPTION
**Pull request by volodymyrss**

### CODE REVIEW

File: tests/test_live.py [hunk 1]
```diff
@@ -220,3 +220,33 @@ def test_large_request(platform):
         E2_keV=200.0,
         scw_list=[f"0665{i:04d}0010.001" for i in range(200)],
     )
+
+@pytest.mark.slow
+#@pytest.mark.parametrize("platform", ["staging-1-3", "staging-1-2", "production-1-2"])
+#def test_pickling(platform):
+def test_pickling():
```
Comment (burnout87) [inline tests/test_live.py:227]: What is this test actually testing?

**Review by burnout87 — COMMENTED** [review]
left a question

File: tests/test_live.py (hunk 1 above)
Comment (volodymyrss) [reply tests/test_live.py]: very peculiar pickling problem! Renamed.

File: tests/test_live.py [hunk 2]
```diff
@@ -222,9 +222,8 @@ def test_large_request(platform):
     )
 
 @pytest.mark.slow
-#@pytest.mark.parametrize("platform", ["staging-1-3", "staging-1-2", "production-1-2"])
-#def test_pickling(platform):
-def test_pickling():
+@pytest.mark.parametrize("platform", ["staging-1-3", "staging-1-2", "production-1-2"])
+def test_peculiar_request_causing_pickling_problem(platform):
```
Comment (burnout87) [inline tests/test_live.py:226]: This test is not actually verifying anything? Is it supposed to be completed?

**Review by burnout87 — COMMENTED** [review]
just a question

File: tests/test_live.py (hunk 2 above)
Comment (volodymyrss) [reply tests/test_live.py:226]: This test is actually verifying very peculiar pickling problem!

Comment (volodymyrss) [reply tests/test_live.py:226]: It is indeed completed.

**Review by burnout87 — APPROVED** [review]

**Pull request merged**